src/tools/kata-ctl/src/main.rs

```diff
@@ -20,7 +21,24 @@ use ops::check_ops::{
     handle_metrics, handle_version,
 };
 
+#[macro_export]
+macro_rules! sl {
+    () => {
+        slog_scope::logger()
+    };
+}
+
 fn real_main() -> Result<()> {
+    //create logger
+    let log_level_name = global_args
+        .value_of("log-level")
+        .ok_or_else(|| anyhow!("cannot get log level"))?;
```

Comment:
For the log level, you'll either have to hard-code it for this PR as something like `let log_level_name = "info"`. Or, better, you could add a `--log-level <level>` command line option. However, to do that will take a bit of reading:

- https://docs.rs/clap/latest/clap/_derive/_tutorial
- https://docs.rs/clap/latest/clap/_derive/index.html

In summary, you'll need to modify `KataCtlCli` in `src/args.rs` to add a `pub LogLevel: String` field as we would want to be able `kata-ctl` like this:

```bash
$ kata-ctl $command ...                     # We currently have this.
$ kata-ctl --log-level $level $command ...  # This is what we'd ideally like to be able to do.
```